tentacle/examples/use_spawn.rs

```diff
@@ -0,0 +1,183 @@
+use bytes::Bytes;
```

Comment:
Suggest to rename this to `simple_using_spawn.rs` and add a line of comment like:

```
/// Implement simple.rs example using `ProtocolSpawn`.

Reply:
This protocol is too simple to demonstrate the advantages of the spawn interface in some scenarios. Could you give a more persuasive example?

Reply:
Its biggest advantage is that it reduces a layer of abstraction, allowing users to master the reading end from a distance. At the same time, the difficulty of using the async environment under callback is eliminated. I personally think that these characteristics can be seen from the simple example. For example, the [`select!`](https://docs.rs/futures/0.3.8/futures/macro.select.html) macro cannot be used under callback